docs/web-ui.md

```diff
@@ -20,32 +20,32 @@ DEV_MODE="true"
 
 ```shell
 ## A semi-colon separated string denoting the allowed domains and/or subdomains. This field is required.
-ALLOWLIST="example.com;foo.bar.com"
+ALLOWLIST="example.com, foo.bar.com"
```

Comment:
This is an unrelated change, but it was bothering me. I don't understand why we changed the default delimeter. @shan4993 do you remember?